### ISSUE TITLE
Decode URL

### ISSUE THREAD

**Issue by tobihagemann** [gnuns/AllOrigins#14]
Since you're suggesting to use `encodeURIComponent()` to encode the URL, it seems that a `decodeURIComponent()` is missing in your code.

https://github.com/gnuns/AllOrigins/blob/0eea4d6e60486376deaf3a812828174490899978/lib/get-page.js#L18

https://github.com/gnuns/AllOrigins/blob/0eea4d6e60486376deaf3a812828174490899978/lib/get-page.js#L41

Instead of unescaping, I'd suggest to replace it with `decodeURIComponent()`.

Btw, I think it totally makes sense to encode the URL in order for URLs like `https://www.google.com/search?q=foo+bar` to work. Otherwise the `+` gets lost. Escaping/unescaping doesn't work for this case.

If you want, I can open a pull request.

Edit: Another suggestion would be to change this line of code:

https://github.com/gnuns/AllOrigins/blob/0eea4d6e60486376deaf3a812828174490899978/lib/index.js#L10

And replace it with:

```js
const page = await getPage(decodeURIComponent(params.url), params.method) 
```

**Comment by gnuns** [comment]
Hi, @tobihagemann, thank you for contributing!
Yeah, it makes sense. If you can open a pull request to solve this, it will be great.

**Comment by tobihagemann** [comment]
Oh snap. I think I made a mistake. 😱 

http://expressjs.com/en/api.html#req.params

> NOTE: Express automatically decodes the values in req.params (using decodeURIComponent).

Sorry I'm new to Express and didn't know that. I guess my change has to be reverted. It "worked" in my project because I did `encodeURIComponent` too often (it seems that `HttpParams` in Angular already does `encodeURIComponent` as well). 😅

I guess `unescape` (which seems to be deprecated) isn't needed either. Sooo I guess removing `decodeURIComponent` here should be sufficient.

https://github.com/gnuns/AllOrigins/blob/e170b6adbb8f538d71356c1e540e5c67f1f5dece/lib/index.js#L10
